spec/file_database_spec.rb

```diff
@@ -93,6 +93,24 @@
     end
   end
 
+  describe '#search' do
+    it 'returns any contacts that matches string given' do
```

Comment:
what happens if no contacts match the string given?
remember to always have tests for the edge cases